gym_electric_motor/physical_systems/electric_motors/squirrel_cage_induction_motor.py

```diff
@@ -150,156 +150,3 @@ def _update_initial_limits(self, nominal_new={}, omega=None):
                                zip(self.FLUXES, flux_alphabeta_limits)}
         flux_nominal_limits.update(nominal_new)
         super()._update_initial_limits(flux_nominal_limits)
-
-
-class DoublyFedInductionMotor(InductionMotor):
-    """
```

Comment:
Is the removal of the DoublyFedInductionMotor intended here?

Reply:
If yes this can be ignored. There is just no mention of it in the commit message, maybe this is a slip up.

Reply:
Yes. As you can see it is in the file "squirrel_cage_induction_motor.py". Somehow, it was a duplicate to the motor defined in "doubly_fed_induction_motor.py". Therefore, I removed it.